test/e2e/scenarios/different-sc-test.go

```diff
@@ -60,21 +60,28 @@ func differentSCTypesTest(driver testsuites.TestDriver) {
 		var (
 			perTestConf *testsuites.PerTestConfig
 			err         error
+			driverType  string
 		)
 		ns = f.Namespace.Name
+		perTestConf, driverCleanup = PrepareCSI(driver, f, "--set driver.drivemgr.amountOfLoopDevices=0")
+
 		if scType == "SSD" {
-			nodes, err := e2enode.GetReadySchedulableNodesOrDie(f.ClientSet)
-			framework.ExpectNoError(err)
-			var nodeNames []string
-			for _, item := range nodes.Items {
-				nodeNames = append(nodeNames, item.Name)
-			}
-			_, err = f.ClientSet.CoreV1().ConfigMaps(ns).Create(constructLoopbackConfigWithDriveType(ns, nodeNames, scType))
-			framework.ExpectNoError(err)
+			driverType = driveTypeSSD
+		} else {
+			driverType = driveTypeHDD
 		}
-		perTestConf, driverCleanup = driver.PrepareTest(f)
 
-		k8sSC = driver.(*baremetalDriver).GetStorageClassWithStorageType(perTestConf, scType)
+		nodes, err := e2enode.GetReadySchedulableNodesOrDie(f.ClientSet)
+		framework.ExpectNoError(err)
+		var nodeNames []string
```

Comment:
The same comment^ ` nodeNames = make([]string, len(nodes.Items))`

Reply:
fixed